sonar-text-plugin/src/main/resources/org/sonar/plugins/secrets/configuration/grafana.yaml

```diff
@@ -0,0 +1,100 @@
+provider:
+  metadata:
+    name: Grafana
+    category: Data visualization
+    message: Make sure this Grafana token gets revoked, changed, and removed from the code.
+  detection:
+    post:
+      patternNot:
+        - "(?i)(?:s|ex)ample|foo|bar|test|abcd|redacted"
+
+  rules:
+    - id: grafana-api-token # Note: deprecated
+      rspecKey: S6762
+      metadata:
+        name: Grafana API tokens should not be disclosed
+      detection:
+        pre:
+          include:
+            content:
+              # This secret pattern checks for the prefix "eyJrIjoi" which is `{"k":"` B64 encoded.
+              # The chance that this can be found in another secret or even normal data is
+              # still somewhat likely, so to be sure we filter for "grafana" here.
+              - grafana
+        matching:
+          pattern: "\\b(eyJrIjoi[a-zA-Z0-9]+(?:={1,2}\\B|\\b))"
```

Comment:
I generated a bunch of secrets and never saw `+` or `/` appear, so I assumed that it could never happen with their current key structure. But it seems to be using [standard B64 encoding](https://github.com/grafana/grafana/blob/5af35f1f3cccf0637c9fc320aaf2e2eddba91726/pkg/components/apikeygen/apikeygen.go#L46C40-L46C40), so I'll add them back.

Reply:
It seem you did not apply the minimum length suggestion. Fine by me but do you have a specific reason not to do so?

Reply:
I just forgot, will fix.